test/core/parser/segments_base_test.py

```diff
@@ -83,25 +91,27 @@ def test__parser__base_segments_base(raw_seg_list, fresh_ansi_dialect):
         (raw_seg_list[0].to_tuple(), raw_seg_list[1].to_tuple()),
     )
     # Check Formatting and Stringification
-    assert str(base_seg) == repr(base_seg) == "<DummySegment: ([3](1, 1, 4))>"
+    assert str(base_seg) == repr(base_seg) == "<DummySegment: ([C:   0, L:  1, P:  1])>"
     assert base_seg.stringify(ident=1, tabsize=2) == (
-        "[3](1, 1, 4)        |  dummy:\n"
-        "[3](1, 1, 4)        |    raw:                                                      'foobar'\n"
-        "[9](1, 1, 10)       |    raw:                                                      '.barfoo'\n"
+        "[C:   0, L:  1, P:  1]|  dummy:\n"
```

Comment:
What does the `C:` value mean?

Reply:
character position in the file. Do you think it adds anything? I could always remove it.

Reply:
It used to be the number in the square brackets of the original position readout. i.e. `[9](1, 1, 10)` would now be `[C:   9, L:  1, P: 10]`, but I'm in two minds about whether anyone cares. I find it a little useful for debugging, but not actually while using it as SQLFluff as a tool. The other option would be to just make it `[L:  1, P: 10]` for simplicity. @barrywhart @NiallRees what do you guys think?

Reply:
I suggest removing it, at least for user-facing output. I can remember very few cases where I was thinking about byte offsets into a text file.